### ISSUE TITLE
Disable linking to non-https images

### ISSUE THREAD

**Issue by lindskogen** [cthit/chalmersit-rails#304]
Prevent security warning in Chrome:

<img width="243" alt="screen shot 2017-09-04 at 11 48 50" src="https://user-images.githubusercontent.com/817046/30021258-09de2a66-9167-11e7-838d-adc2a6b558c5.png">

**Comment by Kalior** [comment]
This would need to include a warning in the post form then

**Comment by lindskogen** [comment]
Yeah, at first I was thinking `s/http/https/` but a warning might be more intuitive for the user.
